ETC.tsv

```diff
@@ -17665,41 +17665,41 @@ ETC_20151224_017666	In substance, this is the kind of Golem and it is a creature
-ETC_20151224_017676	It is not uncommon that the stone statues that are engraved with magical power or parts of architectures move to harm humans. After Medizo Diena, there's no one who thinks that's strange .
-ETC_20151224_017677	Cryomancer Master, Aleister Crowley
-ETC_20151224_017678	Until the year after Medizo Diena, people thought that the monsters that turned from the crops that haven't been harvested would not appear again if harvestors take cautiousness. But, that expectation was gone after the monsters grew by themselves. Pumpleflap is an example of it.
-ETC_20151224_017679	Some places in the devildom are filled with these floating objects in the sky.
-ETC_20151224_017680	Yokotale doesn't walk on it's tale. It is floating in the air and it's inside is empty. But, we don't know if the inside get empty within a short period of time after it's death.
-ETC_20151224_017681	Pyran gets created when the egg of a reptile that is in ashes gets transformed and hatches out.
-ETC_20151224_017682	It is a tool that is used by sorcerer of demons and when the ritual is over, the sorcerer would obtain the ego.
-ETC_20151224_017683	It is a monster that is being created when the tombstone of the dead and the spirit of the grave combine inaccurately.
-ETC_20151224_017684	It is a monster that is being created when a part of the transformed monster from an arthropod gets transformed independently.
-ETC_20151224_017685	Many things were transformed into monsters since Medizo Diena. It is frightening to think about the whereabouts of many anchors which humans dropped into the sea. I just hope that Jurate goddess keep binds them at the sea.
-ETC_20151224_017686	After it became a monster, it was known that it has many uses so they are being hunted by humans. There's a rumor that Winterspoon family buys them at a good price.
-ETC_20151224_017687	It is hard to distinguish them from other monsters that are transformed from other mushroom types, but they are definitely the ones that are transformed from the creatures of the devildom.
-ETC_20151224_017688	In some regions, they created either Kupols or the idols of the spirits. Some of them started to work as evil creatures after Medizo Diena.
+ETC_20151224_017675	A monster that the sorcerers of the devil realm called out using noir. There's a rumor that they use the spirits in the forest after capturing them, but that has yet to be confirmed.
+ETC_20151224_017676	It is not uncommon that the stone statues that are engraved with magical power or parts of architectures move to harm humans. After Medzio Diena, there's no one who think that could be a bit strange.
+ETC_20151224_017677	Cryomancer Master Aleister Crowley
+ETC_20151224_017678	Until the year after Medzio Diena, people thought that the monsters that turned from the crops that haven't been harvested would not appear again if harvestors would be careful. However, that expectation was gone after the monsters grew by themselves. Pumpleflap is an example of this.
```

Comment:
... would not appear again if harvestors would be careful.
... would not appear again if **harvesters were careful**.

and

However, that expectation was gone after the monsters grew by themselves.
However, **they lost their hope when the monsters started growing** by themselves.